corehq/apps/saved_reports/tests/test_models.py

```diff
@@ -21,6 +68,18 @@ def test_report_is_shared_on_domain(self):
         )
         self.config.save()
         self.assertFalse(self.config.is_shared_on_domain())
+        self.config.delete()
```

Comment:
Should this clean up even if the assertion fails?
```suggestion
        self.addCleanup(self.config.delete)
        self.assertFalse(self.config.is_shared_on_domain())
```